src/components/common/SelectInput.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react';
+import { arrayOf, bool, func, string } from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { parseInputErrors } from 'utils/helpers';
+import '../../styles/_variables.scss';
+
+const SelectInput = ({
+  label,
+  name,
+  value,
+  onChange,
+  errors,
+  active,
+  touched,
+  options,
+  ...props
+}) => {
+  // Register field in the form
+  useEffect(() => {
+    onChange({ target: { value } }, true);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
🤔 I think you shouldn't disable this. Maybe your are seeing this warning because you miss `onChange` and `value` as the `useEffect` hook dependencies